src/main/resources/config/liquibase/changelog/20201007105032_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+<changeSet author="krusche" id="20201007105032">
+    <addColumn tableName="course">
+        <column name="registration_confirmation_message" type="VARCHAR(255)"/>
```

Comment:
I haven't seen code which truncates those messages to 255 characters on server side. Otherwise course creation could crash. Validation on client-side should also be done.

Reply:
If this is the case, it might also be better to allow more than 255 characters.

Reply:
Anyways, we still need to add some validation or truncation, otherwise we have a build logs 2.0 case 😄

Reply:
Good point, I added a basic validation